subxt/src/lib.rs

```diff
@@ -65,11 +62,14 @@ mod macros;
 // but leave most types behind their respective modules.
 pub use crate::{
     client::{OfflineClient, OnlineClient},
-    config::{Config, PolkadotConfig, SubstrateConfig},
     error::Error,
-    metadata::Metadata,
 };
 
+// pub use subxt_core::config;
+// pub use subxt_core::config::{Config, PolkadotConfig, SubstrateConfig};
+// pub use subxt_core::dynamic;
+pub use subxt_core::*;
```

Comment:
I'd prefer to be really explicit here so that it's obvious exactly what we're exporting :)